.github/workflows/release.yml

```diff
@@ -242,8 +241,6 @@ jobs:
       - name: Docker meta
         id: meta
         uses: docker/metadata-action@v3
```

Comment:
I think this will fail with `Error: images input required`

Reply:
I have resolved this issue as well as a few others. The rebase logic hosed a bunch of valid configuration in the release.yml github action.